docs/configuration/radio/device.mdx

```diff
@@ -11,24 +11,34 @@ The device config options are: Role, Serial Output, and Debug Log. Device config
+### Roles
+
+| Device Role    | Description                                                                                                              | Best Uses                                                                                                                              |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| CLIENT         | Standard device for communication over the network, supports client apps.                                                | General use for individuals needing to communicate over the Meshtastic network with support for client applications.                   |
+| CLIENT_MUTE    | Same as CLIENT but does not forward packets from other devices.                                                          | Situations where a device needs to participate in the network without assisting in packet routing, reducing network load.              |
+| CLIENT_HIDDEN  | Only broadcasts as needed for stealth or power savings, still rebroadcasts known meshes.                                 | Use in clandestine operations or to reduce airtime/power consumption while still participating in the network.                         |
+| TRACKER        | Broadcasts GPS position packets as priority.                                                                             | Tracking the location of individuals or assets, especially in scenarios where timely and efficient location updates are critical.      |
+| LOST_AND_FOUND | Broadcasts location to default channel regularly for recovery efforts.                                                   | Used for recovery efforts of a lost device.                                                                                            |
+| SENSOR         | Broadcasts priority telemetry packets, optimized for sensor data collection.                                             | Deploying in scenarios where gathering environmental or other sensor data is crucial, with efficient power usage and frequent updates. |
+| TAK            | Optimized for ATAK system communication, reduces routine broadcasts.                                                     | Integration with ATAK systems for communication in tactical or coordinated operations.                                                 |
+| REPEATER       | Dedicated device that focuses solely on extending mesh network coverage by relaying messages. Not visible in Nodes list. | Best positioned in strategic locations to maximize the network's overall coverage not shown in topology.                               |
```

Comment:
maybe "... by relaying messages with minimal overhead" ?

Is "Best positioned in strategic locations to maximize the network's overall coverage not shown in topology." missing punctuation?